packages/dd-trace/test/plugins/externals.json

```diff
@@ -322,7 +322,7 @@
   "mariadb": [
     {
       "name": "mariadb",
-      "versions": ["2.5.6", "3.0.0"]
+      "versions": ["2.5.6", "3.0.0", "3.4.0"]
```

Comment:
This is to force the execution of the tests in 3.4.0, the latest version where `pool.getConnection` returns promises and not a callback.